inc/classes/subscriber/Media/class-webp-subscriber.php

```diff
@@ -377,6 +378,7 @@ public function add_accept_header( $args ) {
 			return $args;
 		}
 
+		$args['headers']['Accept']      = 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8';
 		$args['headers']['HTTP_ACCEPT'] = 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8';
```

Comment:
Do we need this if we have the line above?

Reply:
Yes we need both: https://github.com/wp-media/wp-rocket/blob/5492dcdf444156077eb887c3bcab3df0b73e4c0c/inc/classes/subscriber/Media/class-webp-subscriber.php#L135-L140